hedera-node/src/main/java/com/hedera/services/grpc/ConfigDrivenNettyFactory.java

```diff
@@ -58,7 +58,7 @@ public ConfigDrivenNettyFactory(NodeLocalProperties nodeProperties) {
 
 	@Override
 	public NettyServerBuilder builderFor(int port, boolean sslEnabled) throws FileNotFoundException, SSLException {
-		var activeProfile = nodeProperties.activeProfile();
+		var activeProfile = nodeProperties.nettyMode();
```

Comment:
There used to be a non-standardized `nettyMode` property which can be set to `DEV` or `PROD`, which is different than the active profile. When was this property removed? Now I see that we are adding it back as a standardized `netty.mode` property. The variable here should not be named `activeProfile` as it will give confusion. Also, we will need to update the `hedera-node/configuration` folder and inform dev-ops on those changes.